NuGet.config

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <solution>
+    <add key="disableSourceControlIntegration" value="true" />
+  </solution>
+  <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
+    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="benchmarkdotnet-ci-feed" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />
+    <add key="dotnet-myget-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
is the myget source needed?

Reply:
I believe it is used by something in the benchmarks. I'm not sure if it's needed in the nuget.config if it's in the Directory.Build.props.

Reply:
we need the feed to daily CoreFX and CoreCLR packages. Were they moved from MyGet to `dotnetfeed.blob.core.windows.net` ?